File: ui/src/main/java/com/midtrans/sdk/corekit/models/snap/BankTransferRequestModel.java
package com.midtrans.sdk.corekit.models.snap;

public class BankTransferRequestModel {
    private String vaNumber;

    public BankTransferRequestModel() {}

    public BankTransferRequestModel(String vaNumber) {
        setVaNumber(vaNumber);
    }

    public String getVaNumber() {
        return vaNumber;
    }

    public void setVaNumber(String vaNumber) {
        this.vaNumber = vaNumber;
    }
}

File: corekit/src/main/kotlin/com/midtrans/sdk/corekit/api/model/BankTransferRequestModel.kt
package com.midtrans.sdk.corekit.api.model

import com.google.gson.annotations.SerializedName
import java.io.Serializable

/**
 * Created by rakawm on 4/25/17.
 */
class BankTransferRequestModel : Serializable {
    @SerializedName("va_number")
    var vaNumber: String? = null

    constructor() {}
    constructor(vaNumber: String?) {
        this.vaNumber = vaNumber
    }
}